### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by ABaumher** [FriendsOfGalaxy/galaxy-integration-steam#166] [Ideas]
This is going to be the spot i put some translation related stuff, for now at least. I've attached the initial version of the text we are using. There may be some new strings added as we go. 
[index_html_en-US.txt](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/11443163/index_html_en-US.txt)

**Comment by ABaumher** [comment]
Ideally, we could use whatever language GOG uses for the login page, but i don't think they give us that info. What we're planning to do is add a drop-down list to the top corner of the login page(s) and let you select your language. This information will be saved to the persistent cache, so whenever we lose the authentication for whatever reason (it will expire eventually, and if you change your password it may invalidate), the login page will be in the last language you used.

**Comment by Ricknaark** [comment]
Hi, I have written a translation in french, feel free to use it :))
[index_html_fr.txt](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/11449959/index_html_fr.txt)

**Comment by Zitbvt** [comment]
Bonjour à tous,
En pièce jointe une version traduite en français du fichier txt.

Merci à tous pour votre travail.

Cordialement.
[index_html_fr.txt](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/11450108/index_html_fr.txt)

**Comment by pippo-san** [comment]
Here is the italian translation for you
[index_html_it-IT.txt](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/11556995/index_html_it-IT.txt)
